src/engine/sync/synccontrol.cpp

```diff
@@ -316,15 +330,6 @@ void SyncControl::setInstantaneousBpm(double bpm) {
     m_pBpmControl->setInstantaneousBpm(bpm * m_masterBpmAdjustFactor);
 }
 
-void SyncControl::reportTrackPosition(double fractionalPlaypos) {
-    // If we're close to the end, and master, disable master so we don't stop
-    // the party.
-    if (isMaster(getSyncMode()) &&
-            fractionalPlaypos > kTrackPositionMasterHandoff) {
```

Comment:
`kTrackPositionMasterHandoff` is now unused, please remove the variable

Reply:
done